nextcord/types/interactions.py

```diff
@@ -46,20 +46,26 @@ class _ApplicationCommandOptional(TypedDict, total=False):
     type: ApplicationCommandType
     guild_id: Snowflake
     options: List[ApplicationCommandOption]
-    default_permission: bool
+    dm_permission: bool
+    nsfw: bool
+    default_permission: bool  # This is deprecated, remove it in the future.
+    name_localizations: Dict[str, str]
+    description_localizations: Dict[str, str]
```

Comment:
Don't these localization and permission changes belong in a separate PR?

Reply:
Yes but I asked @alentoghostflame in a thread called ["Support nsfw kwarg for application command"](https://discord.com/channels/881118111967883295/990633056207523860/990658931737129040) in the [Nextcord server](https://discord.com/invite/nextcord) and she said it was fine 🤷‍♀️

Reply:
I see